features/FxA-46-sms-install-link/README.md

```diff
@@ -0,0 +1,54 @@
+# SMS Install Link
+
+https://mozilla.aha.io/features/FXA-46
+
+## Stories
+
+As a user signing up for a Firefox Account, I may want to easily install
+Firefox on my Android or iOS device, removing the friction of searching
+by sending me the link in a text message.
+
+## Success Criteria
+
+This feature will be successful if we measurable increase the rate of
+FIrefox activations for Android and iOS.
+
+A metric for users _trying_ to use the form can be collected, such as an
+event of `fxa.content.event.sms`.
+
+We can optionally also create a metric for the short URL, recording how
+often and what OS visits it. We can additionally create a custom short
+URL for every user (something like https://get.firefox.com/hash(uid)), allowing us to correlate if the user tapped on the
+link after SMS delivery.
+
+We could lastly compare metrics of custom short URLs with their related
+Account to see if the user has successfully connected more devices
+(especially with mobile OSes) to their account.
+
+## Details
+
+On Firefox Desktop, after a user successfully signs up to a new account,
+we will show a small form prompting the user to enter their telephone
+number to be texted a link to install Firefox Mobile.
+
+The SMS capability currently exists in the [Basket API][], allowing us
+to essentially SMS a user via `POST /news/subscribe_sms`.
+
+- UX design of form to show to user
+- Implementation of form in content-server
+- UX of text message wording
+- Short URL pointing to page that redirects based on OS
+  - Play Store for Android
+  - App Store for iOS
+
+## Unresolved Questions
+
+- Should this be an experiment, only being shown to a percentage of
+  users?
```

Comment:
What sort of criteria would be interesting for the experiment? Anything besides `Math.random() < rate`?